packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import { Stack } from '@mui/material'
+import Snackbar from '@mui/material/Snackbar'
+import Alert from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+import IconButton from '@mui/material/IconButton'
+import Icon from 'ui/src/Icon'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+import { Duration } from '@ui-kit/themes/design/0_primitives'
+import { shortenTokenAddress } from 'ui/src/utils/'
+import { useSwitch } from '@ui-kit/hooks/useSwitch'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+type ComponentSize = 'small' | 'medium' | 'large'
+
+type ActionInfoProps = {
+  // Label displayed on the left side
```

Comment:
If you use `/** */` comments instead of `//` you get nice documentation on hover:
![image](https://github.com/user-attachments/assets/a1d8bbe5-86a1-428e-ae36-8bbf146a7d96)